minutes/2019-11-19.md

```diff
@@ -0,0 +1,105 @@
+# Jakarta EE Platform Call
+
+General
+Date: 2019-11-19
+
+Present:
+
+* Ivar Grimstad (Eclipse Foundation)
+* Edwin Derks (Ordina)
+* Kevin Sutter (IBM)
+* Carlos Andres De La Rosa (Eclipse committer)
+* Andy Guibert (IBM)
+* Andy McCright (IBM)
+* Josh Juneau (Independent)
+* Paul Nicolucci (IBM)
+* BJ Hargrave (IBM)
+* Steve Millidge (Payara)
+* Tanja Obradovic (Eclipse Foundation)
+* Rhuan Rocha (Red Hat)
+* Kenji Kazumura (Fujitsu)
+* Martin Stefanko (Red Hat)
+* David Blevins (Tomitribe)
+* Jonathan Gallimore (Tomitribe)
+
+## Agenda
+
+### [KWS] Java SE 8 or Java SE 11 as the minimum. 
+
+Another key decision to allow for continued development efforts.
+* [KWS] Will need to propose a vote on the mailing list indicating the recommendation of Java SE 8 as the minimum, but we would support both Java SE 8 and 11.  (Steve “volunteered” to post the vote.)
+
+### [IG] “Jakartify” the specification documents. Status: https://github.com/eclipse-ee4j/jakartaee-platform/projects/2
```

Comment:
The Status shouldn't be part of the heading.

Reply:
fixed